src/languages/en.ts

```diff
@@ -2515,6 +2531,7 @@ export default {
                 [CONST.INTEGRATION_ENTITY_MAP_TYPES.NOT_IMPORTED]: 'Not imported',
                 [CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE]: 'Not imported',
                 [CONST.INTEGRATION_ENTITY_MAP_TYPES.REPORT_FIELD]: 'Imported as report fields',
+                [CONST.INTEGRATION_ENTITY_MAP_TYPES.NETSUITE_DEFAULT]: 'NetSuite employee default',
```

Comment:
@yuwenmemon @JmillsExpensify @trjExpensify For Xero and other integrations when we have default, we show `Imported`. But design doc mentions `Netsuite employee default`. Can you'll confirm this once?